pkg/virt-controller/watch/clone/clone.go

```diff
@@ -359,7 +359,7 @@ func (ctrl *VMCloneController) createSnapshotFromVm(vmClone *clonev1alpha1.Virtu
 	snapshot := generateSnapshot(vmClone, vm)
 	syncInfo.logger.Infof("creating snapshot %s for clone %s", snapshot.Name, vmClone.Name)
 
-	snapshot, err := ctrl.client.VirtualMachineSnapshot(snapshot.Namespace).Create(context.Background(), snapshot, v1.CreateOptions{})
+	createdSnapshot, err := ctrl.client.VirtualMachineSnapshot(snapshot.Namespace).Create(context.Background(), snapshot, v1.CreateOptions{})
```

Comment:
Please consider the following suggestion:
```go
snapshot := generateSnapshot(vmClone, vm)
createdSnapshot, err := ctrl.client.VirtualMachineSnapshot(snapshot.Namespace).Create(context.Background(), snapshot, v1.CreateOptions{})
if err != nil {
// handle error
}

snapshot = createdSnapshot
```

This way, the `createdSnapshot` is only used temporarily, and there is less potential confusion when to use `snapshot` vs when to use `createdSnapshot`.

Reply:
Thanks! Done